Makefile

```diff
@@ -6,16 +6,18 @@ install:
 	pip install -U -r requirements.txt
 	pip install -U .
 
-.PHONY: isort
-isort:
```

Comment:
I don't think that we need to remove isort and flake8. We need isort at least for right imports ordering. flake8 we need for code linting, black can't help you to show possible bugs.

Reply:
done